XamarinCommunityToolkitSample.UWP/Xamarin.CommunityToolkit.Sample.UWP.csproj

```diff
@@ -12,7 +12,7 @@
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
     <TargetPlatformVersion>10.0.18362.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.16299.0</TargetPlatformMinVersion>
+    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
```

Comment:
Any reason to increase the minVersion?

Reply:
To avoid warning during the build (XF recommends min version 17763)